pkg/worker/v1/rpc/worker.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+import "google/protobuf/timestamp.proto";
+
+// Versioning via package name, for future-proofing
+package wharf.worker.v1;
+
+option go_package = "github.com/iver-wharf/wharf-cmd/pkg/proto/worker/v1";
+
+service Worker {
+  rpc Logs(LogsRequest) returns (stream LogsResponse);
+  rpc StatusEvents(StatusEventsRequest) returns (stream StatusEventsResponse);
+  rpc ArtifactEvents(ArtifactEventsRequest) returns (stream ArtifactEventResponse);
+}
+
+// Empty messages, but exists for potential future usages
+message LogsRequest { }
+message StatusEventsRequest { }
+message ArtifactEventsRequest { }
+
+message LogsResponse {
+  int64 log_id = 1; // not DB value; worker's own ID of the log line
+  int32 step_id = 2; // not DB value; worker's own ID of the step ID
+  google.protobuf.Timestamp timestamp = 3;
+  string line = 4;
+}
```

Comment:
Note this comment: https://github.com/iver-wharf/wharf-cmd/issues/14#issuecomment-1027737336

I really don't think we should be streaming each log line as a separate gRPC message, but instead chunk them.

```proto
service Worker {
  rpc StreamLogs(StreamLogsRequest) returns (stream StreamLogsResponse);
  // ...
}

message StreamLogRequest { }

message StreamLogResponse {
  repeated LogLine logs = 1;
}

message LogLine {
  int64 log_id = 1; // not DB value; worker's own ID of the log line
  int32 step_id = 2; // not DB value; worker's own ID of the step ID
  google.protobuf.Timestamp timestamp = 3;
  string line = 4;
}
```